src/api/QGCOptions.h

```diff
@@ -29,6 +29,9 @@ class QGCOptions : public QObject
     Q_PROPERTY(bool     definesVideo                READ definesVideo               CONSTANT)
     Q_PROPERTY(uint16_t videoUDPPort                READ videoUDPPort               CONSTANT)
     Q_PROPERTY(QString  videoRSTPUrl                READ videoRSTPUrl               CONSTANT)
+    Q_PROPERTY(double   toolbarHeightMultiplier     READ toolbarHeightMultiplier    CONSTANT)
+    Q_PROPERTY(double   defaultFontPointSize        READ defaultFontPointSize       CONSTANT)
+    Q_PROPERTY(bool     enablePlanViewSelector      READ enablePlanViewSelector     CONSTANT)
```

Comment:
This is ok for now I think. But it would be way better for it to be on FirmwarePlugin and on a per-vehicle basis. This ability is something folks have asked for. For example ArduSub doesn't support Missions yet, but it is a superclass of ArduPilot which does support missions. Would be nice if they could turn off Plan when connected to an ArduSub.